### PR TITLE
added option trading

### PR DESCRIPTION
**Pull request by paltryore**
Not sure why GitHub thinks I changed the whole file. Probably in how I updated my fork with your repo. I only added the function trade_option

Examples:
client.trade_option("GOOG", Action.buy, "Call", 945, 170616, 1)
client.trade_option("GOOG", Action.buy, "Put", 932.50, 170616, 10, "Limit", 6.25)
client.trade_option(symbol, ordertype, optiontype, strikeprice, expire_date, quantity, pricetype, targetprice, duration)

Successfully tested option orders with limits, stops, Duration.day_order. Raises exception on error like with client.trade().

Date currently gets entered as YYMMDD (ex. "170616" for 2017-06-16). not sure if good idea.

### CODE REVIEW

**Comment by paltryore** [comment]
I blame my portfolio's poor performance on testing this API.
image

**Comment by kirkthaker** [comment]
I've bought so many random stocks just to test it as well lol. I'll take a look at your pull request and probably merge over the weekend. Thanks for helping.

File: InvestopediaApi/ita.py [hunk 1]
```diff
@@ -1,261 +1,317 @@
...
+        # if a limit or stop order is made, we have to specify the price
+        if price and priceType == "Limit":
+            trade_form.select("input#limitPriceTextBox")[0]["value"] = str(price)
+
+        elif price and priceType == "Stop":
+            trade_form.select("input#stopPriceTextBox")[0]["value"] = str(price)
+
+        prev_page = br.submit(trade_form, trade_page.url)
+        prev_form = prev_page.soup.find("form", {"name": "simTradePreview"})
+        #br.submit(prev_form, prev_page.url)
```
Comment (paltryore) [inline InvestopediaApi/ita.py]: There's an # that shouldn't be in line240 from when I was testing and comparing the two functions.

Comment (kirkthaker) [inline InvestopediaApi/ita.py]: Meaning that I should uncomment the line when I merge, right

Comment (paltryore) [inline InvestopediaApi/ita.py]: Yes.

**Comment by kirkthaker** [comment]
Reviewing & making changes in branch `options`

**Comment by gk159yo** [comment]
@kirkthaker If you are done testing, you should merge.

**Comment by kirkthaker** [comment]
Sorry, this totally slipped my mind! Thanks for helping out

**Pull request merged**